java/spacewalk-java.spec

```diff
@@ -53,97 +51,151 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildArch:      noarch
 ExcludeArch:    ia64 aarch64
 
-Requires:       bcel
-Requires:       c3p0 >= 0.9.1
-Requires:       classpathx-mail
+BuildRequires:  ant
+BuildRequires:  ant-apache-regexp
+BuildRequires:  ant-contrib
+BuildRequires:  ant-junit
+BuildRequires:  ant-nodeps
+BuildRequires:  antlr >= 2.7.6
+BuildRequires:  apache-commons-cli
+BuildRequires:  apache-commons-codec
+BuildRequires:  apache-commons-collections
+BuildRequires:  apache-commons-io
+#??? BuildRequires:  apache-commons-lang
+BuildRequires:  apache-commons-lang3 >= 3.4
+BuildRequires:  apache-commons-logging
+BuildRequires:  bcel
+BuildRequires:  byte-buddy
+BuildRequires:  c3p0 >= 0.9.1
+BuildRequires:  cglib
+%if 0%{?suse_version}
+BuildRequires:  classmate
+BuildRequires:  classpathx-mail
+%endif
+BuildRequires:  concurrent
+BuildRequires:  dom4j
+BuildRequires:  dwr >= 3
+%if 0%{?suse_version}
+BuildRequires:  ehcache >= 2.10.1
+%else
+BuildRequires:  ehcache-core
+%endif
+BuildRequires:  google-gson >= 2.2.4
+BuildRequires:  hibernate-commons-annotations
+BuildRequires:  hibernate5
+BuildRequires:  httpcomponents-asyncclient
+BuildRequires:  httpcomponents-client
+BuildRequires:  jade4j
+BuildRequires:  jaf
 %if 0%{?suse_version}
+BuildRequires:  jakarta-commons-discovery
+BuildRequires:  jakarta-commons-el
+BuildRequires:  jakarta-commons-fileupload
+BuildRequires:  jakarta-commons-validator
+%else
+BuildRequires:  apache-commons-discovery
+BuildRequires:  apache-commons-el
+BuildRequires:  apache-commons-fileupload
+BuildRequires:  apache-commons-validator
+%endif
+BuildRequires:  java-devel >= 11
```

Comment:
For RH/Fedora, this needs to be `java-devel >= 1:11` (for stupid historical reasons related to JPackage, there's _still_ an epoch...)

Reply:
I will change the epoch